internal/storage/bucket_handle_test.go

```diff
@@ -366,6 +367,56 @@ func (t *BucketHandleTest) TestCreateObjectMethodWithGenerationAsZeroWhenObjectA
 	AssertTrue(errors.As(err, &precondition))
 }
 
+func (t *BucketHandleTest) TestCreateChunkUploaderMethodWithValidObject() {
```

Comment:
can we also write a test case or append to this test case where we also call upload and close form chunk uploader created here?

Reply:
Added Upload, Close and corresponding asserts to both the positive tests.